distribution/lib/Standard/Table/0.0.0-dev/src/Io/Delimited_Reader.enso

```diff
@@ -0,0 +1,94 @@
+from Standard.Base import all
+import Standard.Table
+
+import Standard.Base.Error.Extensions as Errors
+from Standard.Base.Error.Problem_Behavior as Problem_Behavior_Module import Problem_Behavior
+from Standard.Table.Error as Table_Errors import Invalid_Row, Mismatched_Quote, Parser_Error, Additional_Invalid_Rows
+from Standard.Base.Data.Text.Encoding as Encoding_Module import Encoding
+from Standard.Table.Io.File_Format import Infer
+
+polyglot java import org.enso.table.read.DelimitedReader
+polyglot java import org.enso.table.read.ParsingFailedException
+polyglot java import org.enso.table.read.InvalidRow
+polyglot java import org.enso.table.read.MismatchedQuote
+polyglot java import org.enso.table.read.AdditionalInvalidRows
+polyglot java import java.lang.IllegalArgumentException
+polyglot java import java.io.IOException
+polyglot java import com.univocity.parsers.common.TextParsingException
+polyglot java import java.io.InputStream
+
+## Reads a delimited file according to the provided format.
+
+   Arguments:
+   - format: The specification of the delimited file format.
+   - file: The file to read.
+   - on_problems: Specifies the behavior when a problem occurs during the
+     operation. By default, a warning is issued, but the operation proceeds.
+     If set to `Report_Error`, the operation fails with a dataflow error.
+     If set to `Ignore`, the operation proceeds without errors or warnings.
+read_file : Delimited -> File -> Problem_Behavior -> Any
```

Comment:
The `Delimited` type is of course defined inside of the `File_Format`, as it is part of this ADT. It can be technically moved, but if I move `Delimited_Reader` to `Internal`, it shouldn't be there because it is a public part of the API. I put it in `File_Format` because all the other formats were kept there.